<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCloudPointSizeSetViewClass</class>
 <widget class="QDialog" name="PointCloudPointSizeSetViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointCloudPointSizeSetView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="PointSize_SpinBox"/>
       </item>
       <item>
        <widget class="QSlider" name="PointSizeHSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
